pyproject.toml

```diff
@@ -246,10 +246,9 @@ docstring-code-format = true
 [tool.pytest.ini_options]
 addopts = "--durations=30 --quiet -r xXs --color=yes --import-mode=importlib"
 filterwarnings = [
-    # NOTE: the last matching option is used
-    "ignore::Warning",               # Ignore all Warning
-    "default::FutureWarning",        # Show FutureWarnings
-    "default::DeprecationWarning",   # Show DeprecationWarnings
+    # NOTE: the LAST matching option would be used
+    "ignore::UserWarning",                 # Ignore UserWarning
+    "error::EncodingWarning:monty",        # Mark `monty` EncodingWarning as error
```

Comment:
TODO: need to test this behaviour as `EncodingWarning` is not built in until Python 3.13